Catappult/appcoins-wallet-android
# Feature/APPC-2806 Transfer NFTs Between Wallets

**What does this PR do?**

   Transfer NFTs between wallets

**How should this be manually tested?**

To test this you will need 2 wallets, an ERC721 NFT, an ERC1155 NFT and to add INFURA_API_KEY_RINKEBY to the Gradle global properties.
You should go to the NFT section of the wallet app and click an NFT if the NFT is ERC721 at the bottom you should seed a button if the NFT is ERC1155 you should not see that button.
Then you should enter your second wallet address and click the send button, if you do not place a valid wallet address and try to send you should be presented with an error message, if everything is ok you should see a success message.

Please test the following cases:
Try to send the same NFT multiple times in a row and see if it present a replacement transaction error (it should present the error)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2806](https://aptoide.atlassian.net/browse/APPC-2806)

**Code Review Checklist**
